### PR TITLE
Accredited Programmes - new user accounts for local development

### PR DESCRIPTION
**Pull request by nsmj63-dxw**
> [See #868 on the Accredited Programmes Trello board.](https://trello.com/c/keEoNI55/868-create-new-new-user-accounts-for-local-development)

https://dsdmoj.atlassian.net/browse/HAAR-1876

### CODE REVIEW

**Comment by nsmj63-dxw** [comment]
Currently WIP: I'd like to check that I'm following the correct approach, and ask for some minor assistance in implementing these roles in the correct manner.

File: src/main/resources/db/dev/data/auth/V900_3__users.sql [hunk 1]
```diff
@@ -325,6 +325,7 @@ INSERT INTO user_role (role_id, user_id) SELECT role_id, user_id from roles, use
 INSERT INTO user_role (role_id, user_id) SELECT role_id, user_id from roles, users where username = 'AUTH_MFA_PREF_EMAIL' and role_code = 'MFA';
 INSERT INTO user_role (role_id, user_id) SELECT role_id, user_id from roles, users where username = 'AUTH_MFA_PREF_EMAIL2' and role_code = 'MFA';
 INSERT INTO user_role (role_id, user_id) SELECT role_id, user_id from roles, users where username = 'AUTH_MFA_PREF_EMAIL3' and role_code = 'MFA';
+INSERT INTO user_role (role_id, user_id) SELECT role_id, user_id from roles, users where username = 'AUTH_MFA_PREF_EMAIL3' and role_code = 'MFA';
```
Comment (ravmoj) [inline src/main/resources/db/dev/data/auth/V900_3__users.sql:328]: duplicate entry

File: src/main/resources/db/dev/data/auth/V900_3__users.sql [hunk 2]
```diff
@@ -443,3 +444,15 @@ VALUES ('E9E6D7E7-D113-43F4-BCF9-0D3AE4A0A8ED', 'ESWE_USER', '{bcrypt}$2a$10$Fmc
 
 -- Roles added to 'HMPPS Eduction Employment UI' User
 INSERT INTO user_role (role_id, user_id) SELECT role_id, user_id from roles, users where username = 'ESWE_USER' and role_code = 'VIEW_PRISONER_DATA';
+
+-- Users used by 'Accredited Programmes UI'
+INSERT INTO users (user_id, username, password, password_expiry, email, first_name, last_name, verified, enabled, locked, secure)
```
Comment (ravmoj) [inline src/main/resources/db/dev/data/auth/V900_3__users.sql:449]: not 'secure' , it's source

File: src/main/resources/db/dev/data/auth/V900_3__users.sql (hunk 1 above)
Comment (nsmj63-dxw) [reply src/main/resources/db/dev/data/auth/V900_3__users.sql:328]: Not a change I made - this should be removed in a separate PR.

File: src/main/resources/db/dev/data/auth/V900_3__users.sql (hunk 2 above)
Comment (nsmj63-dxw) [reply src/main/resources/db/dev/data/auth/V900_3__users.sql]: Fixed, thanks.

File: src/main/resources/db/dev/data/auth/V900_3__users.sql (hunk 1 above)
Comment (nsmj63-dxw) [reply src/main/resources/db/dev/data/auth/V900_3__users.sql:328]: Removed.

File: src/main/resources/db/dev/data/auth/V900_0__clients.sql [hunk 3]
```diff
@@ -30,6 +30,7 @@ VALUES ('another-delete-test-client', 1200, '{}', null, 'client_credentials', 'r
        ('end-date-client', 1200, '{}', null, 'client_credentials', 'read', '$2a$10$lBwbziQlLfiCnn8Kj1PfMujEcLdsJYlYSNJvBRO638gCYTS9yN0xm', null, null, 'read', null),
        ('expired-end-date-client', 1200, '{}', null, 'client_credentials', 'read', '$2a$10$lBwbziQlLfiCnn8Kj1PfMujEcLdsJYlYSNJvBRO638gCYTS9yN0xm', null, null, 'read', null),
        ('hmpps-historical-prisoner-index-client', 3600, '{}', 'ROLE_HISTORICAL_PRISONER_INDEX', 'client_credentials', 'read', '$2a$10$lBwbziQlLfiCnn8Kj1PfMujEcLdsJYlYSNJvBRO638gCYTS9yN0xm', null, null, 'read', null),
+       ('hmpps-accredited-programmes-client-1', 3600, '{}', 'ROLE_VIEW_PRISONER_DATA,ROLE_PRISONER_SEARCH', 'client_credentials', 'read', '$2a$10$lBwbziQlLfiCnn8Kj1PfMujEcLdsJYlYSNJvBRO638gCYTS9yN0xm', 43200, null, 'read', null),
```
Comment (Gweaton) [inline src/main/resources/db/dev/data/auth/V900_0__clients.sql:33]: @ravmoj if you have a moment, would you mind checking this matches the configuration of our dev environment please?

Comment (ravmoj) [reply src/main/resources/db/dev/data/auth/V900_0__clients.sql:33]: @Gweaton , dev is not managed from here... this is purely for local development.

Comment (Gweaton) [reply src/main/resources/db/dev/data/auth/V900_0__clients.sql:33]: @ravmoj sorry if I wasn't clear - we want this local development config to closely mirror our dev config

**Review by ravmoj — APPROVED** [review]
lgtm
